src/tabs.tsx

```diff
@@ -119,8 +119,12 @@ const Tabs: React.FC<TabsProps> = ({selectedIndex, onChange, tabs, dataAttribute
                                         aria-controls={ariaControls}
                                         aria-selected={isSelected ? 'true' : 'false'}
                                     >
-                                        <Inline space={!!icon && !!text ? 8 : 0} alignItems="center">
-                                            {icon && <div className={styles.icon}>{icon}</div>}
+                                        <Inline space={!!Icon && !!text ? 8 : 0} alignItems="center">
+                                            {Icon && (
+                                                <div className={styles.icon}>
```

Comment:
I think we could remove this className and just add the needed dimensions to Icon

Reply:
No, we can't. Svg elements by default have `display: inline-block`, and this would create a space below the icon unless we wrap it inside a div with fixed size (or we set `display: block` in the icon's className).

After discussing offline, we decided to leave the wrapper untouched.

![Screenshot from 2024-08-22 12-08-15](https://github.com/user-attachments/assets/c31998f5-a028-40a0-93aa-ae825f82daa1)

![image](https://github.com/user-attachments/assets/fadc1f6d-a0a4-447f-8d68-4158dca876d8)